File: Bookcase core/src/main/java/cz/vhromada/book/repository/AuthorRepository.java
package cz.vhromada.book.repository;

import cz.vhromada.book.domain.Author;

import org.springframework.data.jpa.repository.JpaRepository;

/**
 * An interface represents repository for authors.
 *
 * @author Vladimir Hromada
 */
public interface AuthorRepository extends JpaRepository<Author, Integer> {
}

File: Bookcase core/src/main/kotlin/cz/vhromada/book/repository/AuthorRepository.kt
package cz.vhromada.book.repository

import cz.vhromada.book.domain.Author
import org.springframework.data.jpa.repository.JpaRepository

/**
 * An interface represents repository for authors.
 *
 * @author Vladimir Hromada
 */
interface AuthorRepository : JpaRepository<Author, Int>
